pages/index.js

```diff
@@ -33,63 +33,48 @@ export default function HomePage() {
               </AmountText>
             </ItemLayout>
           ))}
-        </ListLayout>
+        </ListWrapper>
       )}
-    </>
+      <TabBar />
+    </Wrapper>
   );
 }
 
-const ListLayout = styled.ul`
-  list-style: none;
+const Wrapper = styled.div`
+  display: flex;
+  flex-direction: column;
+  margin-top: 8rem;
+`;
+
+const Subtitle = styled.h2`
+  font-size: var(--fs-subtitle);
+  font-weight: 500;
+`;
+
+const ListWrapper = styled.ol`
   display: flex;
   flex-direction: column;
-  gap: 1em;
-  margin-left: 2em;
-  margin-right: 2em;
+  padding-top: 0.5rem;
+  gap: 2.5rem;
 `;
 
 const ItemLayout = styled.li`
-  background: var(--clr-secondary);
-  border-radius: 0.8em;
   display: flex;
-  flex-direction: row;
   justify-content: space-between;
-  padding-left: 2em;
-  padding-right: 2em;
 `;
 
 const TitleText = styled.h3`
   font-size: var(--fs-caption);
-  text-align: center;
-  color: var(--clr-primary);
+  font-weight: 500;
 `;
 
 const AmountText = styled.p`
   font-size: var(--fs-caption);
-  text-align: center;
-`;
-
-const StyledLink = styled(Link)`
-  background-color: white;
-  border-radius: 3em;
-  border: 2px solid black;
-  padding: 0.7em;
-
-  width: 10em;
-  color: black;
-  display: flex;
-  justify-content: center;
-  margin: auto;
-  font-weight: 700;
 `;
 
 const DetailsLink = styled(Link)`
```

Comment:
if there is no style for the link any more you should use the Link component directly